### PR TITLE
yaraon-blog.com

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/120318

<details><summary>Screenshot:</summary>

Снимок экрана 2022-05-30 в 17 04 58

Снимок экрана 2022-05-30 в 17 05 36

</details><br/>

### CODE REVIEW

File: MobileFilter/sections/specific_web.txt [hunk 1]
```diff
@@ -46,6 +46,9 @@ dailynewsonline.jp,yakuji.co.jp,asahi.com,m.winporn.com,slate.com,sankei.com,gam
 !
 careergarden.jp##.p-ad-wrapper
 sankeibiz.jp###textAd_wpics
+yaraon-blog.com##article > div.widgetarea
```
Comment (Yuki2718) [inline MobileFilter/sections/specific_web.txt:49]: Okay but I prefer `###widgetarea_single_undertopimage_sp`. `###widgetarea_underpager_sp` (top page) is anyway needed as not under `article`. Note `###widgetarea_undercommentform_sp` can not be hidden.

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

File: MobileFilter/sections/specific_web.txt (hunk 1 above)
Comment (Yuki2718) [reply MobileFilter/sections/specific_web.txt:49]: Please change `yaraon-blog.com##article > div.widgetarea` to `yaraon-blog.com###widgetarea_single_undertopimage_sp` and add `yaraon-blog.com###widgetarea_underpager_sp`.

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

**Comment by Yuki2718** [comment]
Optional: `yaraon-blog.com##.searcharea + p + center`

**Review by Yuki2718 — CHANGES_REQUESTED** [review]
Also add `||js.mulan.cloud^` to Japanese adserver for DNS (caught by regex in Base).

**Comment by Yuki2718** [comment]
Please also check on PC, there are some missed ads on left side.

**Review by zloyden — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**